spec/coffee-script-spec.coffee

```diff
@@ -247,3 +247,18 @@ describe "CoffeeScript grammar", ->
     expect(lines[0][0]).toEqual value: '`', scopes: ["source.coffee", "string.quoted.script.coffee", "punctuation.definition.string.begin.coffee"]
     expect(lines[0][1]).toEqual value: 'v', scopes: ["source.coffee", "string.quoted.script.coffee", "constant.character.escape.coffee"]
     expect(lines[1][0]).toEqual value: 'a ', scopes: ["source.coffee", "variable.assignment.coffee", "variable.assignment.coffee"]
+
+  it "tokenizes functions", ->
+    {tokens} = grammar.tokenizeLine("foo = -> 1")
+    expect(tokens[0]).toEqual value: "foo ", scopes: ["source.coffee", "meta.function.coffee", "entity.name.function.coffee"]
+
+    {tokens} = grammar.tokenizeLine("foo bar")
+    expect(tokens[0]).toEqual value: "foo ", scopes: ["source.coffee", "entity.name.function.coffee"]
```

Comment:
`foo bar` is a function call where `foo` is the function and `bar` the argument.